modules/go-deleting-argocd-instance.adoc

```diff
@@ -14,19 +14,19 @@ Delete the Argo CD instances added to the namespace of the GitOps Operator.
 
 [source,terminal]
 ----
-$ oc delete gitopsservice cluster -n openshift-gitops
+$ oc delete gitopsservice cluster
 ----
 
 [NOTE]
 ====
 You cannot delete an Argo CD cluster from the web console UI.
 ====
 
-After the command runs successfully all the Argo CD instances will be deleted from the `openshift-gitops` namespace.
+After the previous command runs successfully the default GitOps Service and its respective Argo CD instance will be deleted from the `openshift-gitops` namespace.
```

Comment:
Maybe we should just say `gitopsservice` instead of `GitOps Service` here on the off chance someone confuses it with the actual [GitOps Service on the appstudio side](https://github.com/redhat-appstudio/managed-gitops#managed-gitops-service-monorepo)

Reply:
Done.